README.md

```diff
@@ -1,56 +1,316 @@
+✓ Creating app/images/apple-touch-icon-144x144.png
+✓ Creating app/images/apple-touch-icon-152x152.png
+✓ Creating app/images/apple-touch-icon-57x57.png
+✓ Creating app/images/apple-touch-icon-60x60.png
+✓ Creating app/images/apple-touch-icon-72x72.png
+✓ Creating app/images/apple-touch-icon-76x76.png
+✓ Creating app/images/favicon-128x128.png
+✓ Creating app/images/favicon-16x16.png
+✓ Creating app/images/favicon-196x196.png
+✓ Creating app/images/favicon-32x32.png
+✓ Creating app/images/favicon-96x96.png
+✓ Creating app/images/mstile-144x144.png
+✓ Creating app/images/mstile-150x150.png
+✓ Creating app/images/mstile-310x150.png
+✓ Creating app/images/mstile-310x310.png
+✓ Creating app/images/mstile-70x70.png
+✓ Creating app/scripts/main.js
+✓ Creating app/scripts/offline-manager.js
+✓ Creating app/styles/stylesheet.css
+
+✓ Creating files… done!
+✓ Installing npm dependencies… done!
+
+Your app has been bootstrapped! Just commit the changes and push the commit
+to the origin/master branch:
+
+git add --all && git commit -m"initial version of Oghliner app"
+git push origin master
+
+Then you can build, offline, and deploy the app using gulp commands.
+
+ℹ For more information about building, offlining and deployment, see:
+    https://mozilla.github.io/oghliner/
+```
+
+Then just commit the changes and push the commit to the origin/master branch:
+
+```
+git add --all && git commit -m"initial version of Oghliner app"
+git push origin master
+```
+
+Note: To bootstrap an app into a different directory than the current one, specify its path when invoking *bootstrap*:
+
+```
+oghliner bootstrap path/to/another/clone/
+```
+
+## Integrate
+
+The *integrate* command adds Oghliner functionality into an existing app. To use it, invoke `oghliner integrate`, passing it the path to the directory containing your app's scripts:
+
+```
+oghliner integrate app/scripts/
+```
+
+Oghliner will copy the *offline-manager.js* script to that directory:
+
+```
+Integrating Oghliner into the app in the current directory…
+
+✓ Copying offline-manager.js to app/scripts/… done!
+
+Oghliner has been integrated into the app!
+
+The app needs to load the script offline-manager.js in order to register
+the service worker that offlines the app. To load the script, add this line
+to the app's HTML page(s)/template(s):
+
+<script src="app/scripts/offline-manager.js"></script>
+
+And commit the changes and push the commit to the origin/master branch:
+
+git commit -m"integrate Oghliner" --all
+git push origin master
+
+Then you can offline and deploy the app using the offline and deploy commands.
+
+ℹ For more information about offlining and deployment, see:
+    https://mozilla.github.io/oghliner/
+```
+
+Then add a &lt;script> tag referencing that script to your app's HTML page(s)/template(s). Oghliner will suggest one, but it doesn't necessarily know your app's directory structure, so make sure it contains the correct path to the file!
+
+## Offline
```

Comment:
I'd specify the `offline` and `deploy` return type (promise), it could be useful for developers.